### ISSUE TITLE
Drawing workbench has empty screen and is unresponsive

### ISSUE THREAD

**Issue by Greenscreener** [cryinkfly/Fusion-360---Linux-Wine-Version-#103]
Hello,

**Describe the bug**
When I create a new drawing, it does get created, but the screen is blank and all of the UI becomes unresponsive to click events (but weirdly not hover events, the UI reacts to hovering on buttons just fine).

**To Reproduce**
1. Open Fusion 360
2. Create an object
3. Create a drawing

**Expected behavior**
Being able to interact with the UI and create the drawing. 

**Screenshots**
![image](https://user-images.githubusercontent.com/22398555/134634947-c482d2e4-83c9-4695-ad9a-a6a8284f7d9e.png)

**Desktop (please complete the following information):**
- Version number of Autodesk Fusion 360 - **2.0.11183**
- Graphics card model + Graphics card driver (Version number) - **Intel i7-3520M + HD Graphics 4000**
- Operating system (OpenSUSE Leap 15.3, Fedora 34, ...) - **OpenSUSE Tumbleweed 20210916**
- Desktop environment (XFCE 4.16, Gnome 40, ...) - **i3**
- DXVK- or OpenGL-Mode - **not sure, used the flatpak installer**

**Additional context**
I used the flatpak installer, but I had to modify some of the commands, because they didn't work properly, but I don't think it was anything functional. I'm not sure if this issue is related to the way it was installed, but I figured this is the best place to get help. 

Thanks in advance.
GS

**Comment by cryinkfly** [comment]
Okay can you download my new script (Flatpak), then I have changed a lot of things! 🙂

**Comment by Greenscreener** [comment]
Sorry for replying this late. 
I used the newest version of the flatpak installer, directly from `main`. I had to go through it and execute all the commands manually, because a lot of them failed (for instance because I don't have the `Downloads` folder). 

But yes, that's it! Somehow, just by opening Fusion360.exe instead of FusionLauncher.exe fixed the problem!
![image](https://user-images.githubusercontent.com/22398555/134864939-b17d1607-8312-47fd-9113-48c4cc960987.png)
Thanks a lot for the help.
